onmt/translate/decode_strategy.py

```diff
@@ -129,6 +133,77 @@ def block_ngram_repeats(self, log_probs):
                 if fail:
                     log_probs[path_idx] = -10e20
 
+    def block_ngram_repeats_efficient(self, log_probs):
```

Comment:
Maybe remove the efficient, since it'll be the only function to do that I reckon.